iron-overlay-manager.html

```diff
@@ -243,7 +267,7 @@
     _getOverlayZ: function(overlay) {
       var z = this._minimumZ;
       if (overlay) {
-        var z1 = Number(window.getComputedStyle(overlay).zIndex);
+        var z1 = Number(overlay.style.zIndex || window.getComputedStyle(overlay).zIndex);
```

Comment:
it happens during the splicing in `addOverlay`. 
Also, using `overlay.style.zIndex` doesn't cause a relayout, so it should be more performant :grin: